### PR TITLE
[FAD-71] My Open Pools - Unverified State

### PR DESCRIPTION
**Pull request by v1n33th**
### Description

Added Open pool component UI

### Checklist

- [x] `CHANGELOG` has been updated (_if appropriate_)
- [x] Environment variables updated (_if appropriate_)

### CODE REVIEW

**Comment by v1n33th** [comment]
Images

Unverified state
Screen Shot 2022-09-09 at 3 33 51 PM

Verified State
Screen Shot 2022-09-09 at 3 34 14 PM

File: packages/client3/components/dashboard/my-open-pool/my-open-pool.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  onButtonClick: () => void;
+};
+
+export function MyOpenPool({ isVerified, onButtonClick }: Props) {
+  return (
+    <div className="grid place-items-center bg-green-100 pt-[60px] pb-[48px]">
+      <BodyText size="normal" className="text-dark-50">
+        {isVerified
+          ? "You haven’t created your pool yet, create your first pool"
+          : "To create the first pool, you need to verify Identity"}
```
Comment (sachasmart-weavik) [inline packages/client3/components/dashboard/my-open-pool/my-open-pool.tsx:14]: Sorry, I know design wrote this. Can you change it to:

"Please create your first pool."

"Please verify your identity to create a pool"

File: packages/client3/components/dashboard/my-open-pool/my-open-pool.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { BodyText, Button } from "../../design-system";
+
+type Props = {
+  isVerified: boolean;
+  onButtonClick: () => void;
+};
+
+export function MyOpenPool({ isVerified, onButtonClick }: Props) {
```
Comment (sachasmart-weavik) [inline packages/client3/components/dashboard/my-open-pool/my-open-pool.tsx:8]: Not sure how I feel about "MyOpenPool"

I would suggest changing it to `ArtristPool` or something.

File: packages/client3/components/dashboard/my-open-pool/my-open-pool.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  return (
+    <div className="grid place-items-center bg-green-100 pt-[60px] pb-[48px]">
```
Comment (sachasmart-weavik) [inline packages/client3/components/dashboard/my-open-pool/my-open-pool.tsx:10]:
```suggestion
    <div className="grid place-items-center bg-green-100 pt-60px pb-48px">
```

:P Ref: https://tailwindcss.com/docs/padding

**Review by sachasmart-weavik — APPROVED** [review]
Super small comments. The naming should probably be a group discussion.

**Review by luansievers — APPROVED** [review]

**Pull request merged**